### ISSUE TITLE
Better documentation :)

### ISSUE THREAD

**Issue by ad1992** [excalidraw/excalidraw#6682]
We launched new [docs](https://docs.excalidraw.com) recently but we need to make it better for user facing queries.

I am listing down the queries which we often get though we have mentioned it in docs but it needs to be improved so its more clear and helps in better communication. Feel free to add more.

- [ ] How to integrate Excalidraw in non React Projects ? Basically this is the way to do it https://docs.excalidraw.com/docs/@excalidraw/excalidraw/integration#browser and we need to improve the docs to communicate this better. There are couple of discussions also opened related to same https://github.com/excalidraw/excalidraw/discussions/4100, https://github.com/excalidraw/excalidraw/discussions/6537,https://github.com/excalidraw/excalidraw/discussions/3849
- [ ] How to support collaboration in Excalidraw npm package ? Its mention in [FAQ](https://docs.excalidraw.com/docs/@excalidraw/excalidraw/faq#does-this-package-support-collaboration-) but we get this query often and we need to improve the docs for the same. There are discussion opened related to this as well (apart from the one linked in FAQ) - https://github.com/excalidraw/excalidraw/discussions/6676, https://github.com/excalidraw/excalidraw/discussions/5902, https://github.com/excalidraw/excalidraw/discussions/3941, https://github.com/excalidraw/excalidraw/discussions/4924, https://github.com/excalidraw/excalidraw/discussions/4685

**Comment by LuluNwenyi** [comment]
Hi @ad1992 I'd love to work on this issue. Any ideas on how to go about this?? I'd like to know what tool is used for the documentation, so I can know how to proceed.

**Comment by tristinDLC** [comment]
@LuluNwenyi The Excalidraw dev docs are built using [Docusaurus](https://docusaurus.io/) which is a great documentation framework developed by Facebook.

**@excalidraw-team**, I've worked with numerous docs projects with the bulk of them being Docusaurus builds (especially if I was allowed to chose the platform I got to deploy with). Among many hats, I wear the technical writer one at my current job for all our user guides, our internal-use knowledge base, and our API developer docs. I'll let the above commenter take the project in the OP, but I'll offer my services anywhere else in the docs if needed. 

I'd love to be able to play around with it and see what all can be enhanced to provide as nice of a DX as possible for anyone needing to read any docs... let me know

**Comment by ad1992** [comment]
@LuluNwenyi yep we are using [Docusaurus](https://docusaurus.io/) for writing the docs. As mentioned in description, there are few areas where documentation needs to be improved to make it more intutive. So you can start to pick from that list.

Also you can checkout the docs and suggest where you think docs needs to be improved as well.

@tristinDLC sure would appreciate your suggestions as well as we improve the docs :)

**Comment by tristinDLC** [comment]
I missed the notification that I was tagged here by _quite_ some time haha. I'm definitely still interested in taking a look and seeing what improvements can be made. 

Once again, sorry for massively delays response @ad1992

**Comment by AmanSoni1-apex** [comment]
Hi! I’d like to contribute to improving the documentation.

I can start with:
1. Adding a `CODE_OF_CONDUCT.md`
2. Creating a new docs page for “Using Excalidraw without React (Vanilla JS integration)”
3. Improving onboarding/README sections if needed

Please let me know if this direction is okay.
If yes, I’ll start working on a PR.
